firmware/console/binary/tunerstudio.cpp

```diff
@@ -277,6 +277,28 @@ static void onlineApplyWorkingCopyBytes(int currentPageId, uint32_t offset, int
 	}
 }
 
+/**
+ * Read internal structure for Live Doc
+ * This is somewhat similar to read page and somewhat similar to read outputs
+ * We can later consider combining this functionality
+ */
+static void handleGetStructContent(ts_channel_s *tsChannel, int structId, int size) {
+	tsState.readPageCommandsCounter++;
+
+	const void *addr = NULL;
+	if (structId == LDS_CLT_INDEX) {
```

Comment:
switch instead of `if`? (switch can give far better performance when checking consecutive numbers, it will use a lookup table)

Reply:
Maybe probably. There is a chance this would be code-generated later or soon.